python/cuml/test/test_pickle.py

```diff
@@ -597,3 +598,29 @@ def assert_model(pickled_model, X):
         assert state["_fit_status_"] == 0
 
     pickle_save_load(tmpdir, create_mod, assert_model)
+
+
+def test_small_rf(tmpdir):
+
+    result = {}
+
+    def create_mod():
+        n_samples = 100
```

Comment:
I think it would be nice to have variables passed as unit_param pytest parameters.

Reply:
@Salonijain27 What does unit_param do?

Reply:
it only uses the parameters for running the unit test and not for stress or quality test